### PR TITLE
GenesisProtocol -threshold function receive paramHash (instead of proposalId)

### LINKED ISSUE #500: Don't require passing in a proposalId to GenesisProtocol.threshold
The current threshold should be the same for the whole DAO, not per proposal. Can't we track that param by the avatar so I don't have to look up the new value for every proposal after every stake?

### PR DESCRIPTION
**Pull request by orenyodfat**
resolve #500 

Fix warning from IntVoteIntraface

### CODE REVIEW

File: contracts/VotingMachines/IntVoteInterface.sol [hunk 1]
```diff
@@ -34,26 +34,26 @@ interface IntVoteInterface {
         ) external returns(bytes32);
 
     // Only owned proposals and only the owner:
-    function cancelProposal(bytes32 _proposalId) external onlyProposalOwner(_proposalId) votable(_proposalId) returns(bool);
```
Comment (dkent600) [inline contracts/VotingMachines/IntVoteInterface.sol]: What are the use cases that motivate removing `onlyProposalOwner` and `votable` in all these cases?

File: contracts/VotingMachines/AbsoluteVote.sol [hunk 2]
```diff
@@ -300,6 +285,31 @@ contract AbsoluteVote is IntVoteInterface {
...
     }
 
+    /**
+      * @dev execute check if the proposal has been decided, and if so, execute the proposal
+      * @param _proposalId the id of the proposal
+      * @return bool true - the proposal has been executed
+      *              false - otherwise.
+     */
+    function _execute(bytes32 _proposalId) internal votable(_proposalId) returns(bool) {
+        Proposal storage proposal = proposals[_proposalId];
```
Comment (dev-matan-tsuberi) [inline contracts/VotingMachines/AbsoluteVote.sol]: Why this change?

**Comment by orenyodfat** [comment]
execute function should be external as it appears in the interface though
in the other end it suppose to be called internally .
This modification is to make the compiler happy as it was complaining about
that .

**Comment by orenyodfat** [comment]
Empty modifiers are ignored .
The compiler was complaining about it .

File: contracts/VotingMachines/IntVoteInterface.sol (hunk 1 above)
Comment (orenyodfat) [reply contracts/VotingMachines/IntVoteInterface.sol]: Empty modifiers are ignored .
The compiler was complaining about it .
